packages/amplify-graphql-schema-generator/src/ts-schema-generator/generate-ts-schema.ts

```diff
@@ -0,0 +1,15 @@
+import * as ts from 'typescript';
+import { Schema } from '../schema-representation';
+import { createImportExpression, createSchema } from './helpers';
+
+const file = ts.createSourceFile('schema.ts', '', ts.ScriptTarget.ESNext, false, ts.ScriptKind.TS);
+const printer = ts.createPrinter({ newLine: ts.NewLineKind.LineFeed });
+
+export const generateTypescriptDataSchema = (schema: Schema): string => {
+  const result = printer.printList(
+    ts.ListFormat.MultiLine,
+    ts.factory.createNodeArray([createImportExpression(), ts.factory.createIdentifier('\n'), createSchema(schema)]),
```

Comment:
Do we need an eslint ignore comment?

Reply:
Looks okay, eslint didn't complain about this.

Reply:
Sorry, I mean should we add an eslint ignore comment to the generated output. A customer may some additional rules.

Reply:
Assuming you mean "may add some additional _eslint_ rules" I agree. We don't want our generated code causing problems in a customer project.

Reply:
Added eslint-disable comment to the generated schema.